packages/code/src/Code.tsx

```diff
@@ -288,14 +274,22 @@ function Code({
     },
     className,
     getScrollShadowStyle(scrollState, mode),
+    { [singleLineWrapperStyle]: !isMultiline },
+    {
+      [css`
```

Comment:
we can inline this, but it's enough lines that it wouldn't hurt to pull into it's own variable

Reply:
I don't feel strongly about this either way. Done!